Upstream-Status: Pending

Fix obsolete settings with RPM5.

The following patch is from Michael Schroeder <mls@suse.de>

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

diff -ur git.orig/src/pool.c git/src/pool.c
--- git.orig/src/pool.c	2011-03-25 15:49:34.885834194 -0500
+++ git/src/pool.c	2011-03-28 10:55:13.138786524 -0500
@@ -70,6 +70,10 @@
   pool->disttype = DISTTYPE_DEB;
 # endif
 #endif
+#ifdef RPM5
+  pool->obsoleteusesprovides = 1;
+  pool->implicitobsoleteusesprovides = 1;
+#endif
   return pool;
 }
 
